### DISCUSSION TITLE
Hoe werkt de ManningResistance?

### DISCUSSION THREAD

**Discussion by tessa-andringa-sweco** [Deltares/Ribasim#500] [Q&A] [answered]
We zien dat er alleen een node_id, length, manning_n, profile_width en profile_slope opgegeven moet worden. Maar aangezien je in de Manningformule een nat oppervlak/omtrek nodig hebt, hadden wij ook verwacht dat je een bodemhoogte of waterdiepte mee zou moeten geven. Hoe zit dit precies?

**Comment by Huite** [comment, marked as answer]
Hallo Tessa, de beschrijving is hier te vinden:

https://deltares.github.io/Ribasim/core/equations.html#manning-connection

Daar zou je wijzer uit moeten worden, als dat niet zo is, dan moeten we het aanvullen.

De ManningResistance haalt zijn waterpeil uit het peil dat door Ribasim gesimuleerd wordt. Ook de bodemhoogte komt uit de gegevens van de Basins aan weerzijde. Er valt te bediscussieren of dat een te sterke overversimpeling is die effectief gebruik hindert -- het vervelende met meerdere bodemhoogtes (basin links, basin rechts, aanknoping links, aanknoping rechts) is dat het makkelijk wordt om verwarrende, inconsistente waardes op te geven.
